go.mod

```diff
@@ -1,10 +1,8 @@
 module github.com/stolostron/mtv-integrations
 
-go 1.24.0
+go 1.25.0
 
-toolchain go1.24.3
-
-godebug default=go1.24
+godebug default=go1.25
```

Comment:
I don't think this line is necessary? I'm not familiar with `godebug`, but I suspect the `godebug default` for this module file aligns with the `go` declaration.
```suggestion
```